src/index.unit.test.js

```diff
@@ -229,6 +229,20 @@ describe('Unit tests', () => {
       });
     });
 
+    describe('getUpdates', () => {
+      it('get all updates since the given date', async () => {
```

Comment:
I should've been doing this with my existing tests, but the way I think it's supposed to be structured is something like

```javascript
describe('#getUpdates', () => {
  describe('when a timestamp is specified', () => {
    it('makes request with specified timestamp', () => {
      // test
    });
  describe('when a timestamp is not specified', () => {
    it('makes request with empty string', () => {
      // another test
    });
  });
} 
```

This way, each nested `describe` illustrates a specific case and the `it`s identify the expected behavior.